src/testCombiner.ts

```diff
@@ -120,12 +121,21 @@ export function genTestClass(
     annotation = shortenClassNames({ imports: testImports, body: annotation });
   }
 
-  let rule = '@org.junit.Rule';
-  if (testImports && testImports.length) {
-    if (testImports.indexOf('org.junit.Rule') === -1) {
-      testImports.push('org.junit.Rule');
-    }
-    rule = '@Rule';
+  if (classRules.size > 0 && (testImports.indexOf('org.junit.Rule') === -1)) {
```

Comment:
Not sure if this is necessary 
```suggestion
  if (classRules.size > 0 && testImports.indexOf('org.junit.Rule') === -1) {
```